tests/test_odl.py

```diff
@@ -195,7 +201,10 @@ def test_checkin_already_fulfilled(self):
         })
 
         self.api.queue_response(200, content=lsd)
-        self.api.checkin(self.patron, "pin", self.pool)
+        assert_raises(
+            CannotReturn, self.api.checkin,
+            self.patron, "pin", self.pool,
+        )
```

Comment:
If you agree with what I say in https://github.com/NYPL-Simplified/circulation/pull/879/files#r173530170 then I imagine both cases would be tested here.